build/scripts/crossgen.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+set -e
+
+BIN_DIR="$( cd $1 && pwd )"
```

Comment:
It would consider the home directory to be the target directory. It fails when crossgen starts looking for mscorlib.

I didn't consider this script to be worth hardening, but I can add guard checks if you think it's worth it.